keras_nlp/models/deberta/deberta_presets.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+backbone_presets = {
+    "deberta_xsmall": {
+        "config": {
+            "vocabulary_size": 128100,
+            "num_layers": 12,
+            "num_heads": 6,
+            "hidden_dim": 384,
+            "intermediate_dim": 1536,
+            "dropout": 0.1,
+            "max_sequence_length": 512,
+            "bucket_size": 256,
+        },
+        "preprocessor_config": {},
+        "description": (
+            "XSmall size of DeBERTa (v3). "
```

Comment:
Write out Extra Small. And I would just write DeBERTaV3, as that's how they say it in the paper.